build.gradle

```
@@ -60,11 +60,80 @@ pluginBundle {
     website = url
     vcsUrl = url
     description = project.description
-    tags = ['marytts', 'text-to-speech', 'tts', 'speech synthesis']
+    tags = ['marytts', 'plugin', 'text-to-speech', 'tts', 'speech synthesis']
```

I don't think these tags gets updated after the plugin is first added to the Gradle Plugins Portal.
But more to the point, tagging a plugin with 'plugin' is probably redundant.
```suggestion
    tags = ['marytts', 'text-to-speech', 'tts', 'speech synthesis']
```